src/extension.ts

```diff
@@ -32,6 +38,13 @@ export async function activate(context: ExtensionContext): Promise<void> {
     manifests.forEach((manifest) => store.manifestFound(manifest))
     //TODO: allow the user to select a manifest
     const manifest = manifests[0]
+    if (!versionCompare(flatpakVersion, manifest.requiredVersion)) {
+      statusBarItem.setStatus({
```

Comment:
The status bar item currently is used for flatpakRunner's status, while the new statusBarItem introduced in https://github.com/bilelmoussaoui/flatpak-vscode/pull/71 is for activeManifest's status. It could be a notification for now and change it later

Reply:
Switched to an error dialog and left a todo comment for future iterations